src/frontend/screens/Onboarding/walletSelection/screens/import/components/ImportAndCreateOptions/index.module.scss

```diff
@@ -0,0 +1,12 @@
+@use '../../mixins';
```

Comment:
We don't have alias in place? Optional case

Reply:
i think it is fine to have relative path here